src/coreclr/nativeaot/BuildIntegration/BuildIntegration.proj

```diff
@@ -9,7 +11,12 @@
     <Content Include="*.*" Exclude="$(MSBuildProjectFile)" />
   </ItemGroup>
 
-  <Target Name="Build">
+  <!-- TODO-LLVM: This is needed only until we ship Microsoft.NETCore.App.Ref containing the right version of generator -->
```

Comment:
```suggestion
  <!-- TODO-LLVM-Upstream: <new issue link>. -->
```
Could you open a new issue about this?

<details>
<summary>TODO-LLVM-Upstream?</summary>

We use this to tag places that either:
1) Represent concessions in code readability with the goal of conflict avoidance.
2) Represent technical debt (i. e. workarounds) due to the downstream status.

The difference with a plain `TODO-LLVM` is that `TODO-LLVM`s can be addressed downstream.
</details>